discord/types/message.py

```diff
@@ -108,7 +108,7 @@ class _MessageOptional(TypedDict, total=False):
     components: List[Component]
 
 
-MessageType = Literal[0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 14, 15, 18, 19, 20, 21]
+MessageType = Literal[0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 14, 15, 18, 19, 20, 21, 22, 23]
```

Comment:
What do types 22 and 23 refer to in regards to a message?

Reply:
They're documented in the MessageTypes enum.